pkg/config/config.go

```diff
@@ -141,3 +156,41 @@ func (t *ThrottlingConfig) Interval() time.Duration {
 func (c *Config) Rules() []*autoheal.HealingRule {
 	return c.rules
 }
+
+// OpenChangeObserver open the change obeserver channels
+//
+func (c *Config) OpenChangeObserver() {
+	c.configFilesObserver = new(observer.Observer)
+	c.configFilesObserver.Open()
+}
+
+// CloseChangeObserver close the change obeserver channels
+//
+func (c *Config) CloseChangeObserver() {
```

Comment:
I think that now this is never called. Do we still need it?

Reply:
I had a bad rebase ... it is called in the healer `defer` to program exit

fixing the rebase now ... lots of code missing in healer ...